src/feature/category/ui/CategoryTab.tsx

```
@@ -0,0 +1,41 @@
+    <div
+      className="w-full px-3 md:justify-center justify-start flex gap-4 overflow-x-scroll whitespace-nowrap"
+      style={{ scrollbarWidth: "none" }}
+    >
+      <CategoryTabItem href={`?categoryId=${0}`} active={isAllSelected}>
+        전체
+      </CategoryTabItem>
+      {categories.map((c) => {
+        const isSelected = c.id === selectedCategoryId;
+        return (
+          <CategoryTabItem href={`?categoryId=${c.id}`} active={isSelected}>
+            {c.name}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)